src/Bases/ModelBase.py

```diff
@@ -108,7 +108,9 @@ def setVariable(self, name, value):
     """
     ret = None
     # multi-entry or single-entry?
-    if len(value) == 1:
+    if isinstance(value, str):
+      ret = value.split()[0]
```

Comment:
The assumption here is: if the developer provided a string, we will just pick the first key word, and do ```setattr(self, name, ret)```. Personally, I would like the developer just define the self variable directly, something like ```self.name=ret``` in their code, and not use setVariable method.

Reply:
oki, let me see if my edits are looking better